addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/UsernameEnumerationScanRule.java

```diff
@@ -402,9 +394,10 @@ public void scan() {
 
                 // get a random user name the same length as the original!
                 String invalidUsername =
-                        RandomStringUtils.random(
-                                currentHtmlParameter.getValue().length(), RANDOM_USERNAME_CHARS);
-                log.debug("The invalid username chosen was [{}]", invalidUsername);
+                        RandomStringUtils.randomAlphabetic(
```

Comment:
`randomAlphabetic` includes uppercase alphabets as well. If we are okay with that then it should be noted in the changelog.

Reply:
I'll `toLowerCase` it in order to maintain the same behavior. It could make a difference on some servers/targets.